packages/ui-extensions/src/surfaces/customer-account/api/docs.ts

```diff
@@ -117,6 +117,12 @@ export interface Docs_ResourceItem_Button_Action
     | 'kind'
   > {}
 
+export interface Docs_Menu_Button_Action
+  extends Omit<
```

Comment:
Using Omit reflects how [our HOC works](https://github.com/Shopify/checkout-web/blob/main/packages/checkout-web-ui/src/components/Button/withContextualStyle.tsx#L23). We don't pick props, we force the values of a very specific set of props (those are omitted cause you can set another value, except for apparence which is not omitted). Every other props will be passed to Button. 

The day Checkout removes `submit` prop from Button, the doc will update and won't show that prop in Menu as well since we rely on the same type, so I believe it's okay to keep it for now.